### ISSUE TITLE
Automatically insert argument to "--order" into the "fields" list, if it's not already present

### ISSUE THREAD

**Issue by bskinn** [ofek/pypinfo#10]
If `pypinfo` is called with an `--order` argument, but the corresponding field is not present in the list of fields provided after the desired package name, the BigQuery API complains:

```
>pypinfo -o date numpy
Traceback (most recent call last):
...
google.api.core.exceptions.BadRequest: 400 POST https://www.googleapis.com/bigquery/v2/projects/pypi
nfo-btskinn/queries: SELECT clause has mix of aggregations 'download_count' and fields 'download_dat
e' without GROUP BY clause
```

Whereas everything's fine when the field is explicitly passed:

```
>pypinfo -o date numpy date
download_date download_count
------------- --------------
2017-09-28    136028
2017-09-27    88843
2017-09-26    94603
2017-09-25    93221
2017-09-24    61966
2017-09-23    64148
2017-09-22    95677
2017-09-21    97871
2017-09-20    93427
2017-09-19    87520
2017-09-18    93313
2017-09-17    62543
2017-09-16    63271
2017-09-15    80160
2017-09-14    92697
2017-09-13    90869
2017-09-12    79222
2017-09-11    65286
2017-09-10    57141
2017-09-09    47484
```

Perhaps a check should be added for the presence of the `--order` argument in the `fields` list, with the sort field added to `parsed_fields` if it's not there? (Probably right after [this](https://github.com/ofek/pypinfo/blob/e61a81db994f3e04ad8b7f67e291ebe01cf69e6b/pypinfo/cli.py#L76) would be good.)

**Comment by ofek** [comment]
Fixed in 4.0.0, well done!

**Comment by bskinn** [comment]
Oh... this is actually a separate issue from the field name translation addressed by the pull request. Fixing this, if desired, would involve actively injecting the field name of the `--order` cl arg into the list of fields to display. I wasn't sure if this would be a desired
behavior so I opened an issue on it.

**Comment by bskinn** [comment]
@ofek Thanks! This is my first accepted PR with real code involved so I'm pretty excited to see it merged. :-)

**Comment by ofek** [comment]
Sorry, re-opened! I'll check this out Monday.

**Comment by ofek** [comment]
Fixed in `10.0.0`!
